sdk/spring-sdk/src/test/java/kalix/springsdk/testmodels/subscriptions/PubSubTestModels.java

```diff
@@ -232,14 +232,16 @@ public Effect<String> transform(EmployeeEmailUpdated emailUpdated) {
 
     }
 
-    @Subscribe.EventSourcedEntity(EmployeeEntity.class)
```

Comment:
This test model had two subscriptions. That's currently not legal, but we don't have proper validation yet. 

I have been experimenting with some validation, but will PR next week (hopefully).

Reply:
added https://github.com/lightbend/kalix/issues/7912 for visibility